frameworks/cassandra/docs/limitations.md

```diff
@@ -5,7 +5,19 @@ feature_maturity: preview
 enterprise: 'no'
 ---
 
-* Data Center Name
+## Rack-Aware Placement
+
+The DC/OS Cassandra Service does not currently leverage rack information for placement decisions.  However, placement across racks can be performed manually using placement constraints; application-level configuration will still not be possible.   
+
+## Security
+
+The security features are not supported at this time.
```

Comment:
Same as above ... "Cassandra's native TLS, authentication, and authorization features are not supported at this time."